### PR TITLE
Added subcriber verified scope and already subscribed msg

### PR DESCRIPTION
**Pull request by joecohens**
//cc @cachethq/owners 

Closes https://github.com/cachethq/Cachet/issues/1210

### CODE REVIEW

File: app/Handlers/Events/Incident/SendIncidentEmailNotificationHandler.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function handle(IncidentWasReportedEvent $event)
 
         // Only send emails for public incidents.
         if ($event->incident->visible === 1) {
-            foreach ($this->subscriber->all() as $subscriber) {
+            foreach ($this->subscriber->verified()->get() as $subscriber) {
```
Comment (GrahamCampbell) [inline app/Handlers/Events/Incident/SendIncidentEmailNotificationHandler.php]: missing brackets on `subscriber`

File: app/Handlers/Events/Incident/SendMaintenanceEmailNotificationHandler.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ public function handle(MaintenanceWasScheduledEvent $event)
 
         $data = AutoPresenter::decorate($event->incident);
 
-        foreach ($this->subscriber->all() as $subscriber) {
+        foreach ($this->subscriber->verified()->get() as $subscriber) {
```
Comment (GrahamCampbell) [inline app/Handlers/Events/Incident/SendMaintenanceEmailNotificationHandler.php]: same

File: app/Models/Subscriber.php [hunk 3]
```diff
@@ -63,11 +63,21 @@ public static function boot()
     }
 
     /**
+     * Scope a query to only include verified subscribers.
+     *
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeVerified($query)
```
Comment (GrahamCampbell) [inline app/Models/Subscriber.php]: missing typehint

File: app/Models/Subscriber.php [hunk 4]
```diff
@@ -63,11 +63,21 @@ public static function boot()
     }
 
     /**
+     * Scope a query to only include verified subscribers.
+     *
```
Comment (GrahamCampbell) [inline app/Models/Subscriber.php]: missing param annotation :ghost:

Comment (joecohens) [inline app/Models/Subscriber.php]: :stuck_out_tongue:

**Pull request merged**